scripts/import_map/update.ts

```diff
@@ -0,0 +1,18 @@
+import { parse } from "https://deno.land/std/flags/mod.ts";
```

Comment:
We should hardcode the version for this import.

nit: Should this come from dev_deps.ts as well?

Reply:
Good catch 💯 

I don't think `dev_deps.ts` will work well here since each script operates independently and do not behave as a unified module